oshi-core/src/main/java/oshi/software/os/linux/LinuxOSProcess.java

```diff
@@ -339,6 +344,40 @@ public boolean updateAttributes() {
         return true;
     }
 
+    /**
+     * If some details couldn't be read from ProcPath.PID_STATUS try reading it from
+     * ProcPath.PID_STAT
+     *
+     * @param status
+     *            status map to fill.
+     * @param stat
+     *            string to read from.
+     */
+    private static void getMissingDetails(Map<String, String> status, String stat) {
+        if (status == null || stat == null) {
+            return;
+        }
+
+        int nameStart = stat.indexOf("(");
+        int nameEnd = stat.indexOf(")");
+        if (Util.isBlank(status.get("Name")) && nameStart < nameEnd) {
```

Comment:
I believe the strict `nameStart < nameEnd` should catch the case where there's no `( ... name ... )` in the String.  However, just to be completely 100% sure here, we should probably ensure `nameStart > 0`.

Reply:
If there is no name in the string, both would be index of -1 and therefore the check would turn to false.
I will add the extra check, but can't imagine a case where its necessary.

Reply:
Added the check